content/02.body.md

```diff
@@ -0,0 +1,282 @@
+
+![Individual node degree by sampling method. In the left column, 70% of literature edges were sampled with uniform probability. In the right column, literature derived degrees are over or under the corresponding systematically-derived degree.](https://github.com/greenelab/xswap-analysis/raw/bd40eb244ffe37454dc08099f5742ef3e962c70d/img/7_well_studied_degree_bias/hexbin.png){#fig:sampling_vs_systematic width="85%"}
+
+Networks created through degree-unbiased experimentation have node degrees that are linearly correlated with the number of true relationships.
+Many domains, however, do not have unbiased experimental methods, either because such methods have not yet been developed or because such methods are not possible, in principle.
+For example, compound-disease treatment relationships in human patients can not be elucidated via large scale, unbiased experimentation.
+Without unbiased data, the available networks on which analysis methods can be applied (the data themselves) have node degree bias of an unquantifyable magnitude.
+It is therefore helpful to use methods that can quantify and correct for the effects of degree on the methods used for network analysis, inference, and prediction.
+
+### Unsupervised link prediction
+
+The application motivating the present work is (biomedical) network link prediction.
+In particular, we restrict our attention to unsupervised prediction, meaning that there are no edges which are known but not already in the network.
+Supervised network reconstruction tasks are useful for many methods, especially for evaluating and comparing prediction methods.
+However, reconstruction tasks are inappropriate for training or evaluating prediction methods between networks with very different degree distributions.
+Corrupted (or "sampled") networks share their degree distributions, roughly, with the uncorrupted networks (for example, see Supplemental figure {@fig:supp-dists}).
+The issue remains, though, that the target networks (or the networks whose edges are the true goal of modelling and prediction) may have vastly different degree distributions compared to the networks available.
+Because the degree distributions of networks can be highly biased when compared to the distribution of true relationships, it is imperative to take into account the differences between degree distribution when generalizing a trained supervised prediction method to new data.
```

Comment:
I got to roughly this point in the paragraph and got confused about the topic of it. You might read this out loud and see what you think. It feels like it could probably use a bit of a restructuring.